packages/grid/_modules_/grid/locales/heIL.ts

```diff
@@ -90,18 +90,18 @@ const heILGrid: Partial<GridLocaleText> = {
     count !== 1 ? `${count.toLocaleString()} שורות נבחרו` : `שורה אחת נבחרה`,
 
   // Total rows footer text
-  footerTotalRows: 'סך הכל שורות: ',
```

Comment:
Are you sure you want to remove the space after `":"`?

Reply:
Yes, I looked at other translations and it is the case in all of them